### PR TITLE
fix favicon

### LINKED ISSUE #93: [frontend] add favicon

### PR DESCRIPTION
**Pull request by kittybest**
close #93 .

### CODE REVIEW

**Review by vivianjeng — CHANGES_REQUESTED** [review]
You should change the favicon path in `webpack.config.js`
(I am not sure the `webpack.worker.js` is used for)
But favicon should be previewed in cloudflare environment
like this
https://34c87077.trusted-setup-favicon.pages.dev/

**Comment by vivianjeng** [comment]
And I prefer favicon to be like this
截圖 2023-09-22 上午10 56 42
@ChialiT ?

**Comment by ChialiT** [comment]
> And I prefer favicon to be like this 截圖 2023-09-22 上午10 56 42 @ChialiT ?

This one is better, stand out more.

**Comment by kittybest** [comment]
> You should change the favicon path in `webpack.config.js` (I am not sure the `webpack.worker.js` is used for) But favicon should be previewed in cloudflare environment like this https://34c87077.trusted-setup-favicon.pages.dev/

That's my fault, I just saw I modified the wrong file 😅

**Comment by kittybest** [comment]
image

This is favicon downloaded from upper comment(with black background). 

Or are you saying the one without background color? only with the unirep logo in blue?

image

like this.

**Comment by vivianjeng** [comment]
I think with black background

**Comment by kittybest** [comment]
then @ChialiT could you give me a `.svg` version of this logo with black background? I only have the screenshot, and can only find the one without any background in figma.

**Comment by ChialiT** [comment]
> then @ChialiT could you give me a `.svg` version of this logo with black background? I only have the screenshot, and can only find the one without any background in figma.

tag you in the figma

**Comment by kittybest** [comment]
image
It looks good in search bar in safari browser, but

image
is added white border while on tab in safari browser.

Meanwhile, under chrome browser seems good.

File: packages/frontend/webpack.worker.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const path = require('path')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const CssMinimizerPlugin = require('css-minimizer-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
```
Comment (vivianjeng) [inline packages/frontend/webpack.worker.js]: I think it is not used in this file?

**Review by vivianjeng — APPROVED** [review]

**Pull request merged**